describe the HTTP method config (#1769)
iterative/dvc.org: content/docs/command-reference/remote/modify.md

    to the remote in the form: `custom_auth_header: password`.
    `custom_auth_header` and `password` (or `ask_password`) parameters should
    also be configured.

>   - `method` - override the HTTP method to use for file uploads (ex. PUT). Default: POST

Reviewer: Minor correction to comply with the style:
```suggestion
  - `method` - override [the HTTP method](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods)
    to use for file uploads (e.g. `PUT`/`POST`). By default, `POST` is used.
```

    to the remote in the form: `custom_auth_header: password`.
    `custom_auth_header` and `password` (or `ask_password`) parameters should
    also be configured.

>   - `method` - override [the HTTP method](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods)

Reviewer: ```suggestion
  - `method` - override
    [the HTTP method](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods)
```

>     to the remote in the form: `custom_auth_header: password`.

Reviewer 2: ```suggestion
    to the remote in the form: `custom_auth_header: <value>`.
```

Reviewer 2: UNRELATED: @efiop what do you think about changing this BTW ^, using `password` as value here seems confusing, since `password` is also a field you can set separately of `auth`. Unless I'm confused and that's the point...

Reviewer: That's precisely what you should do: set password in the config. There is a sentence clarifying that right after this one. Using `<value>` would be confusing here, though.

Reviewer 2: OK thanks. Leaving this as is then.

    `custom_auth_header` and `password` (or `ask_password`) parameters should
    also be configured.

>   - `method` - override
>     [the HTTP method](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods)

Reviewer 2: ```suggestion
  - `method` - override the
    [HTTP method](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods)
```

>     to use for file uploads (e.g. `PUT`/`POST`). By default, `POST` is used.

Reviewer: Also, might want to mention that PUT could be used with Artifactory. People will find it through search, which should be useful.

>     to use for file uploads (e.g. `PUT`/`POST`). By default, `POST` is used.

Reviewer 2: I'm not sure I get how this works. Can you guys provide a quick example? What this describes so far is:

```console
$ dvc remote modify  myremote auth method
```

How do you set the actual method? Why is the method a type of auth? Thanks

Reviewer: > $ dvc remote modify myremote method PUT

Reviewer 2: OK so this bullet shouldn't be nested under `auth` then 🙂

Reviewer 2: Or are both needed?

Reviewer: Oops, didn't notice that. Good catch! Thanks!

>     [HTTP method](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods)
>     to use for file uploads (e.g. `PUT`/`POST`). By default, `POST` is used.

Reviewer: ```suggestion
    [HTTP method](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods) to
    use for file uploads (e.g. `PUT`/`POST`). By default, `POST` is used.
```

>     use for file uploads (e.g. `PUT`/`POST`). By default, `POST` is used.

Reviewer: ```suggestion
    use for file uploads (e.g. `PUT` should be used for
    [Artifactory](https://www.jfrog.com/confluence/display/JFROG/Artifactory+REST+API)).
    By default, `POST` is used.
```

  $ dvc remote modify myremote auth basic
  ```

- `method` - override the
  [HTTP method](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods) to
  use for file uploads (e.g. `PUT` should be used for
  [Artifactory](https://www.jfrog.com/confluence/display/JFROG/Artifactory+REST+API)).

>   By default, `POST` is used.
> 

Reviewer 2: ````suggestion
  By default, `POST` is used.

  ```dvc
  $ dvc remote modify myremote method PUT
  ```
````
